server/api/core/core.js

```diff
@@ -975,53 +978,5 @@ export default {
     const version = packageJson.version;
     Logger.info(`Reaction Version: ${version}`);
     Shops.update({}, { $set: { appVersion: version } }, { multi: true });
-  },
-
-  /**
-   * @summary Method for getting all schemas attached to a given collection
-   * @deprecated by simpl-schema
-   * @private
-   * @todo TODO: Remove collectionSchema method in favor of simpl-schema
-   * @name collectionSchema
-   * @param  {string} collection The mongo collection to get schemas for
-   * @param  {Object} [selector] Optional selector for multi schema collections
-   * @return {Object} Returns a simpleSchema that is a combination of all schemas
-   *                  that have been attached to the collection or false if
-   *                  the collection or schema could not be found
-   */
-  collectionSchema(collection, selector) {
```

Comment:
Not used anymore

Reply:
Can we provide any sort of deprecation path there rather than just removing this in case application developers are using it?